README.md

```
@@ -1,209 +1,204 @@
+### Upgrading
+When upgrading an existing installation, reload systemd service definitions and restart `asusd`:
 ```sh
 systemctl daemon-reload && systemctl restart asusd
 ```
+Alternatively, reboot the system to apply updates.
 
-## Uninstalling
-
-Run `sudo make uninstall` in the source repo, and remove `/etc/asusd/`.
-If you have installed with a package manager, use your package managers uninstall function.
+### Uninstalling
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)